src/gui/FileDialog.h

```diff
@@ -36,6 +36,7 @@ class FileDialog
     QString getExistingDirectory(QWidget* parent = nullptr, const QString& caption = QString(),
                                  QString dir = QString(), QFileDialog::Options options = QFileDialog::ShowDirsOnly);
 
+    void setNextForgetDialog();
```

Comment:
I would change this to a simple `setRememberLastDir()` setter and rename the member variable to `m_rememberLastDir`.

Reply:
The thing is that since it's a static instance it will need to reset the member variable anyway, otherwise once set all the future dialog will be set as well until it re-set

Reply:
Then at least rename it to (m_)forgetLastDir and reverse the logic. The current name is a bit confusing.